MAVProxy/modules/mavproxy_locdb.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+'''locationdb command handling'''
+
+import struct
+from pymavlink import mavutil
+from MAVProxy.modules.lib import mp_module
+
+from MAVProxy.modules.lib import mp_module
+from MAVProxy.modules.lib import mp_settings
+from MAVProxy.modules.lib import mp_util
+
+if mp_util.has_wxpython:
+    from MAVProxy.modules.lib import mp_menu
+    from MAVProxy.modules.mavproxy_map import mp_slipmap
+
+
+
+class DBItem():
+    def __init__(self):
+        self.key = 0
+        self.timestamp_ms = 0
+        self.lat = 0
+        self.lon = 0
+        self.alt = 0
+        self.velx = 0
+        self.vely = 0
+        self.velz = 0
+        self.accx = 0
+        self.accy = 0
+        self.accz = 0
+        self.heading = 0
+        self.radius = 0
+        self.populated_fields = 0
+        self.onMap = False
+
+class LocDBModule(mp_module.MPModule):
+    def __init__(self, mpstate):
+        super(LocDBModule, self).__init__(mpstate, "locdb", "LocationDB handling", public = True)
+        self.add_command('locdb', self.cmd_locdb,       'locationdb management',
+                         ["<list>"])
+        self.locdb_ftp_name = "@LOCATIONDB/locationdb.dat"
+        self.num_items = 0
+        self.background_fetch = True
+        self.item_refresh_timer = mavutil.periodic_event(0.5)
+        self.dbitems = []
+
+    def cmd_locdb(self, args):
+        '''locationdb commands'''
+        usage = "usage: locdb <list>"
+        if len(args) < 1:
+            print(usage)
+            return
+        
+        if args[0] == "list":
+            self.background_fetch = False
+            self.dbitems_list()
+        else:
+            print(usage)
+
+    def dbitems_list(self):
+        print("Total DB Items: %u" % self.num_items)
+        for item in self.dbitems:
+            print("Key:%s, Last Update:%u, Lat:%.2f, Lon:%.2f, Alt:%.2f , Vel:(%.2f, %.2f, %.2f), Acc:(%.2f, %.2f, %.2f), Hdg:%.2f, Rad:%.2f" % (item.key,
+                                                                                                                                                 item.timestamp_ms,
```

Comment:
Put a carriage return after the open parenthesis to tidy this (i.e. wrap onto a fresh line so the stuff isn't massively indented)

Reply:
Fixed it.